docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,29 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
 5. any Persistent Disks associated with your cluster
 
 While we hope this list is comprehensive, there could be some lingering resources used by Google Cloud and not deleted automatically that we're not aware of.
+
+Benchmarking Figures from DeepCell Kiosk Paper
+---------------------------------------------------------
+
+The DeepCell Kiosk paper (Bannon et al., 2020) presents cost and runtime benchmarks for running a generic segmentation pipeline on datasets of given sizes inside the DeepCell Kiosk with different sets of constraints (Fig. 1b).
```

Comment:
I think this should be "Benchmarking the cluster" and then later on how one can use the output files to recreate the graphs.

> The DeepCell Kiosk comes with a utility for benchmarking the scalability and performance of a deep learning workflow. To run your own benchmarking, please read below.

Reply:
I did this. But I realize now that I forgot to add in the figure recreation section at the end. I'll push that in a few minutes.

Reply:
Pushed.